Replicator/RevFinder.cc

```diff
@@ -217,6 +217,7 @@ namespace litecore::repl {
                 auto mode = (deletion < 4) ? RevocationMode::kRevokedAccess
                                            : RevocationMode::kRemovedFromChannel;
                 revoked.emplace_back(new RevToInsert(docID, mode));
+                sequences.push_back({RemoteSequence(change[0]), 0});
```

Comment:
This line shouldn't be added -- the sequences array is parallel with docIDs and revIDs. It isn't used for revoked revs.

Reply:
The reason to add the sequence with body size = 0 here is to allow the checkpoint to be updated for the revoked revisions. Is there an alternative way for this?

Reply:
From the Slack conversation with @snej, this solution is OK.